backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/sales/invoice/report_details_local_tax_reporting.jrxml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Jaspersoft Studio version 6.5.1.final using JasperReports Library version 6.5.1  -->
```

Comment:
I think you shall add this in main report for sales invoice and check that in general, if not local currency is set, nothing is shown in that report.
Otherwise, next will create another report.